profiles/common/modules/features/nexteuropa_core/nexteuropa_core.tokens.inc

```diff
@@ -27,13 +27,12 @@ function nexteuropa_core_tokens($type, $tokens, array $data = array(), array $op
     $node = $data['node'];
     foreach ($tokens as $name => $original) {
       if (strstr($name, ':parents-uri') !== FALSE) {
-        list($field_name, $delta,) = explode(':', $name);
+        list($field_name) = explode(':', $name);
         $field_name = str_replace('-', '_', $field_name);
         if (isset($node->{$field_name})) {
           $node_wrapper = entity_metadata_wrapper('node', $node);
-          $terms = $node_wrapper->{$field_name}->value();
-          if (isset($terms[$delta - 1])) {
-            $term = $terms[$delta - 1];
+          $term = $node_wrapper->{$field_name}->value();
```

Comment:
$node_wrapper->{$field_name}->value() can return an array if the field is multi-values (see last part of the ["Working with lists" section](https://www.drupal.org/docs/7/api/entity-api/entity-metadata-wrappers#example-value-set-save). Should we take it into account?